docs/tutorials/lyla-jr.md

```diff
@@ -0,0 +1,594 @@
+# Lyla and Ev's Snack Chase
+### @explicitHints true
+### @flyoutOnly true
+
+
+## Lyla and Ev's Snack Chase @showdialog
+
+**Let's code a video game with Lyla, Everett and Stu!**
+
+![Pickle Party Game](/static/tutorials/lyla/lyla.gif "Characters chasing pickle slices." )
+
+
+
+## {Step 5}
+
+**We need snacks to chase!**
+
+![Add a flying pickle with a new block](/static/tutorials/lyla/pickle.gif "Choose the pickle from the gallery" )
+
+#### ~ tutorialhint
+
+```blocks
+//@highlight
+sprites.sendFlying(lyla_imgs.picklechip, 1)
+```
+
+
+
+## {Step 7}
+
+**Add a background!**
+
+![Use the set scene block](/static/tutorials/lyla/bg.gif "Use the set scene block" )
+
+The background in the block is Loops Lunch! Click on the image to draw your own background.
+
+
+
+#### ~ tutorialhint
+
+```blocks
+sprites.sendFlying(lyla_imgs.picklechip, 1)
+//@highlight
+scene.setBG(lyla_imgs.diner)
+```
+
+
+
+
+## {Step 6}
+
+**Try your game!**
+
+[Re-Link Intro Video Here]
```

Comment:
Is this a placeholder you meant to update?

Reply:
It's a placeholder until we get the video. Not exactly sure when we'll get it, but if it's going to take too long, I'll make a stand-in.